addon/components/docs-viewer/x-nav-item/template.hbs

```diff
@@ -5,15 +5,15 @@
       activeClass='docs-text-brand docs-font-medium'
       data-test-id='nav-item'
       data-test-label=label}}
-      {{label}}
+      {{this.label}}
```

Comment:
Looking at the component, `label` is a positional param, so I think if this is a passed in thing, it should be `@label` rather than `this.label`